swarm/storage/chunker_test.go

```diff
@@ -263,7 +174,7 @@ func TestSha3ForCorrectness(t *testing.T) {
 
 }
 
-func TestDataAppend(t *testing.T) {
+func XTestDataAppend(t *testing.T) {
```

Comment:
@nonsense If you use `vim` that's not an issue ;)

I won't argue convention, because I don't know. However, what I like about `t.Skip()` is that you get told which tests are skipped and (hopefully) a reason for it when you try to run them.

Reply:
You are right, I haven't noticed that with `XTest` tests do not appear with a `Skip` when running `go test -v`.

Adding a reason inside t.Skip() would definitely be nice, I had to comment on this ticket when I saw that this test is disabled.

In this case I would be happy to follow one convention or the other, depending how we agree as a team ;)